server/lib/db.js

```diff
@@ -319,6 +320,75 @@ const saveReplayMetadata = (metadata) => new Promise((resolve, reject) => {
         .catch((error) => reject(error));
 });
 
+/**
+ * Creates session using a webId.
+ * Returns session secret or undefined if something went wrong
+ */
+const createSession = async (webId) => {
+    // Find user
+    const user = await getUserByWebId(webId);
+    if (user === undefined || user === null) {
+        // TODO: create user in users collection. Use Ubi API to fill fields
+        console.log(`No user found for id ${webId}`);
+        return undefined;
+    }
+
+    // Create session
+    const sessions = db.collection('sessions');
+    const sessionId = uuid();
+    await sessions.insertOne({
+        sessionId,
```

Comment:
Not something we have to do right now, but I wonder if we should store the hash of the secret instead of the real session. Obviously that means we'd also have to change the validation when requests come in - let me know what you think.